t/t0401-post-command-hook.sh

```diff
@@ -80,16 +80,12 @@ test_expect_success 'with post-index-change config' '
 
 	# add keeps the worktree the same, so does not run post-command.
 	# and this should work through an alias.
-	rm -f post-command.out post-index-change.out &&
 	git config alias.dd add &&
+	rm -f post-command.out post-index-change.out &&
```

Comment:
This line should probably be added to the correct spot already in the preceding commit.